label/label_blackbox_test.go

```diff
@@ -103,6 +105,59 @@ func (s *TestLabelRepository) TestCreateLabelWithWrongColorCode() {
 	assert.Contains(s.T(), err.Error(), "labels_background_color_check")
 }
 
+func (s *TestLabelRepository) TestSaveLabel() {
```

Comment:
Please rename to `TestSave` and use sub tests here for each scenario.

Reply:
Fixed. 9945b9941f1241159878fc83faeb0a83ca037f9b